### DISCUSSION TITLE
When data is ingested into my node registry, is it integrated with the rest of the PDS?

### DISCUSSION THREAD

**Discussion by jordanpadams** [NASA-PDS/registry#90] [Q&A] [answered]
Is there a distinction between a Discipline Node’s individual registry and the PDS-wide registry.  If so, which is the one into which we have ingested our data?  

In other words:  Have we now created our node registry?  And is it integrated into the PDS-wide registry?  Or are you not making that distinction anymore?

**Comment by pds-ops** [comment, marked as answer]
## Short Answer
Once you ingest your data into "your" registry, it is fully-integrated with all PDS registries, and accessible to the world through the PDS Search API alongside all the other node data at https://pds.nasa.gov/api/search/.

## Long Answer

### Current Design
Each node does have their own distinct Registry Service running through AWS OpenSearch, which are then integrated using OpenSearch Cross-cluster Search (CCS) through the EN's Central Registry.

For more details on the architecture, see the [Registry About page](https://nasa-pds.github.io/registry/about.html) and/or the [Cloud Architecture page](https://nasa-pds.github.io/registry/cloud/architecture.html).

### Future Design
We are currently investigating a refactoring of the Registry architecture to utilize a [multi-tenancy architectural approach through Amazon OpenSearch Service](https://docs.aws.amazon.com/prescriptive-guidance/latest/patterns/build-a-multi-tenant-serverless-architecture-in-amazon-opensearch-service.html). This may impact some of the functions for interacting with node-specific registries, but will _NOT_ require nodes to re-load any existing data.
